lte/gateway/python/magma/mobilityd/subscriberdb_client.py

```diff
@@ -14,18 +14,37 @@
 import ipaddress
 from typing import Optional
 from lte.protos.subscriberdb_pb2 import APNConfiguration
+from magma.subscriberdb.sid import SIDUtils
 
 import grpc
 import logging
 
-from magma.subscriberdb.sid import SIDUtils
+
+class StaticIPInfo:
+    """
+    Operator can configure Static GW IP and MAC.
+    This would be used by AGW services to generate networking
+    configuration.
+    """
+
+    def __init__(self, ip: str, gw_ip: str, gw_mac: str,
+                 vlan: str):
+        self.ip = ipaddress.ip_address(ip)
+        self.gw_mac = gw_mac
+        gw_ip_parsed = None
+        try:
+            gw_ip_parsed = ipaddress.ip_address(gw_ip)
+        except ValueError:
+            logging.debug("invalid internet gw ip: %s", gw_ip)
```

Comment:
do we want to do something in addition to log the invalid msg? it seems that in this case `self.gw_ip` would be None

Reply:
If this fails AGW would flood SGI network with UE packet.
This can be check looking at mobilityd GW info table.